E2E-TESTING.md

```diff
@@ -43,12 +45,25 @@ You need to do this every time you install a new version of Xcode.
 ### Usage
 
```

Comment:
Looks great, just small thing I'm missing here - big info that those tasks take lot of time to run :-)